### PR TITLE
[FR] Update `Judgement/osu!catch` and `Judgement/osu!taiko`

### PR DESCRIPTION
**Pull request by DoryanR**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Judgement/osu!taiko/fr.md [hunk 1]
```diff
@@ -1,34 +1,57 @@
...
+
+La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
+
+L'erreur de hit est arrondie et les valeurs maximales d'erreur de hit sont arrondie à l'entier le plus proche, ce qui signifie que pour les GREAT et les OK, les fenêtres peuvent être jusqu'à 1,5 ms plus courtes des deux côtés, tandis que les fenêtres de miss peuvent être jusqu'à 0,5 ms plus courtes ou plus large des deux côtés, par rapport à ce que les formules suggèrent.
+
+## Mécaniques de jugement
+
+### Notes petites/larges
+
+Les petites et les notes larges sont jugées par un GREAT, un OK ou un MISS en fonction de la précision avec laquelle elles sont hit. Hit une note avant la fenêtre MISS n'a aucun effet, et ne pas frapper une note entraînera un MISS une fois la fenêtre MEH passée. Le fait de hit la mauvaise touche pour la couleur de la note entraînera également un MISS.
```
Comment (DoryanR) [inline wiki/Gameplay/Judgement/osu!taiko/fr.md:21]:
```suggestion
Les petites et les notes larges sont jugées par un GREAT, un OK ou un MISS en fonction de la précision avec laquelle elles sont hit. Hit une note avant la fenêtre MISS n'a aucun effet, et ne pas frapper une note entraînera un MISS une fois la fenêtre MEH passée. Le fait de hit la mauvaise touche pour la couleur de la note entraînera également un MISS.
```

@Walavouchey does the MEH judgement exist on osu!taiko? I'm not sure, as there's no mention of it in the tables.

File: wiki/Gameplay/Judgement/osu!taiko/fr.md [hunk 2]
```diff
@@ -1,34 +1,57 @@
...
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!taiko) | [Précision](/wiki/Gameplay/Accuracy#osu!taiko) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit300g.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit300.png) | GREAT | 300 | 100% | `50 - 3 × OD` |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit100k.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit100.png) | OK | 150 | 50% | `120 - 8 × OD` si OD ≤ 5, et `110 - 6 × OD` si OD ≥ 5 |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit0.png) | MISS | 0 | 0% | `135 - 8 × OD` si OD ≤ 5, et `120 - 5 × OD` si OD ≥ 5 |
+
+La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!taiko/fr.md:13]: Je pense qu'il faudrait rajouter le terme "overall difficulty" entre parenthèses car il n'est pas traduit dans l'article fr et ça a plus de sens pour son abréviation OD.

File: wiki/Gameplay/Judgement/osu!taiko/fr.md [hunk 3]
```diff
@@ -1,34 +1,57 @@
...
+
+L'erreur de hit est arrondie et les valeurs maximales d'erreur de hit sont arrondie à l'entier le plus proche, ce qui signifie que pour les GREAT et les OK, les fenêtres peuvent être jusqu'à 1,5 ms plus courtes des deux côtés, tandis que les fenêtres de miss peuvent être jusqu'à 0,5 ms plus courtes ou plus large des deux côtés, par rapport à ce que les formules suggèrent.
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!taiko/fr.md:15]:
```suggestion
L'erreur de hit est arrondie et les valeurs maximales d'erreur de hit sont arrondies à l'entier le plus proche, ce qui signifie que pour les GREAT et les OK, les fenêtres peuvent être jusqu'à 1,5 ms plus courtes des deux côtés, tandis que les fenêtres de miss peuvent être jusqu'à 0,5 ms plus courtes ou plus large des deux côtés, par rapport à ce que les formules suggèrent.
```

File: wiki/Gameplay/Judgement/osu!taiko/fr.md [hunk 4]
```diff
@@ -1,34 +1,57 @@
...
+
+Les petites et les notes larges sont jugées par un GREAT, un OK ou un MISS en fonction de la précision avec laquelle elles sont hit. Hit une note avant la fenêtre MISS n'a aucun effet, et ne pas frapper une note entraînera un MISS une fois la fenêtre MEH passée. Le fait de hit la mauvaise touche pour la couleur de la note entraînera également un MISS.
+
+Les notes larges peuvent être hit avec deux touches de la bonne couleur en même temps (à moins de 30 ms l'une de l'autre), ce qui double le score.
+
+### Drum rolls
+
+Les drum rolls donnent 300 points (360 pendant le [kiai time](/wiki/Gameplay/Kiai_time)), tandis que les large drum rolls donnent 720 points (864 pendant le kiai time), par tick de drum roll effectué dans le bon timing.
+
+Un hit trop rapide ou trop lent empêchera la collecte des ticks. Les limites sont environ de hit deux fois plus vite que les ticks apparaissent et de hit plus lentement que tout les 5 ticks.
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!taiko/fr.md:29]:
```suggestion
Un hit trop rapide ou trop lent empêchera la collecte des ticks. Les limites des hits sont à peu près deux fois plus rapides que l'apparition des ticks, et plus lentes que tous les 5 ticks.
```

File: wiki/Gameplay/Judgement/osu!taiko/fr.md (hunk 2 above)
Comment (DoryanR) [inline wiki/Gameplay/Judgement/osu!taiko/fr.md:13]:
```suggestion
La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) (overall difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
```

File: wiki/Gameplay/Judgement/osu!taiko/fr.md [hunk 5]
```diff
@@ -1,34 +1,57 @@
...
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!taiko) | [Précision](/wiki/Gameplay/Accuracy#osu!taiko) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit300g.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit300.png) | GREAT | 300 | 100% | `50 - 3 × OD` |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit100k.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit100.png) | OK | 150 | 50% | `120 - 8 × OD` si OD ≤ 5, et `110 - 6 × OD` si OD ≥ 5 |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit0.png) | MISS | 0 | 0% | `135 - 8 × OD` si OD ≤ 5, et `120 - 5 × OD` si OD ≥ 5 |
+
+La fenêtre de hit dépend de la [difficulté globale (OD)](/wiki/Beatmap/Overall_difficulty) (overall difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
```
Comment (DoryanR) [inline wiki/Gameplay/Judgement/osu!taiko/fr.md:13]:
```suggestion
La fenêtre de hit dépend de la [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
```

File: wiki/Gameplay/Judgement/osu!taiko/fr.md [hunk 6]
```diff
@@ -1,34 +1,57 @@
...
+
+| Image | Nom | [Valeur de hit](/wiki/Gameplay/Score/ScoreV1/osu!taiko) | [Précision](/wiki/Gameplay/Accuracy#osu!taiko) | Erreur de hit maximum (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit300g.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit300.png) | GREAT | 300 | 100% | `50 - 3 × OD` |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit100k.png) ![](/wiki/shared/judgement/osu!taiko/taiko-hit100.png) | OK | 150 | 50% | `120 - 8 × OD` si OD ≤ 5, et `110 - 6 × OD` si OD ≥ 5 |
+| ![](/wiki/shared/judgement/osu!taiko/taiko-hit0.png) | MISS | 0 | 0% | `135 - 8 × OD` si OD ≤ 5, et `120 - 5 × OD` si OD ≥ 5 |
+
+La fenêtre de hit dépend de la [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
```
Comment (Biokax) [inline wiki/Gameplay/Judgement/osu!taiko/fr.md:13]:
```suggestion
La fenêtre de hit dépend de l'[overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) de la beatmap. Un hit est considéré à l'intérieur d'une fenêtre de hit si `erreur de hit < erreur de hit maximum`, ce qui signifie que la valeur indiquée est égale à la moitié de la largeur de la fenêtre. La fenêtre MISS compare par exception `erreur de hit ≤ erreur de hit maximum` à la place.
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**